types/redux-form/lib/SubmissionError.d.ts

```diff
@@ -2,6 +2,7 @@ import { FormErrors } from "../index";
 
 export declare class SubmissionError<FormData = {}, ErrorType = string> extends Error {
   constructor(errors?: FormErrors<FormData, ErrorType>);
+  public errors: FormErrors<FormData, ErrorType> | undefined;
```

Comment:
[Travis reported](https://travis-ci.org/DefinitelyTyped/DefinitelyTyped/builds/569426943#L369): `member-access  'public' is implicit.` Please fix.
Also in my opinion it should not be nullable: from [source code](https://github.com/erikras/redux-form/blob/master/src/SubmissionError.js) the `errors` member is always set up.

Reply:
In this case the constructor maybe also needs fixing.

Reply:
I made the changes requested.  The 'public' was required when I did it locally be linter/typescript - however, I removed it as requested.